channels.js

```diff
@@ -0,0 +1,40 @@
+require('dotenv').config({silent: true});
+
+const ds = require('./datastore.js');
+const core = require('./core.js');
+
+var channelStore = new ds.DataStore();
+setInterval(getChannels, 10000);
```

Comment:
one nice tip for handling time amounts is to break it up in to their parts

so you can write `10 * 1000` to make it easier to understand this is 10 seconds